doc/src/vpr/command_line_usage.rst

```diff
@@ -1816,6 +1816,24 @@ The following options are used to enable power estimation in VPR.
 
     Instructions on generating this file are provided in :ref:`power_estimation`.
 
+Server Mode Options
+^^^^^^^^^^^^^^^^^^^^^^^^
+The following options are used to enable server mode in VPR.
```

Comment:
Can you explain a bit more about why someone would use server mode in VPR and what it does?  E.g., "If vpr is in server mode it listens on a socket for commands from a client; currently this is used to enable interactive timing analysis and visualization of timing paths in the vpr UI under control of a separate client."

Reply:
I took your description from example, if you don't mind, i doubt i could write better.
solved https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/6eb326eff3a779b5208099de2ad6ad4eddff05e2